CellProfiler.py

```diff
@@ -0,0 +1,3 @@
+from cellprofiler import __main__
```

Comment:
We should use:

```Python
import cellprofiler.__main__
```

per [docs.python-guide.org](http://docs.python-guide.org/en/latest/writing/structure/#modules).